doc/subsystems/test/sanitycheck.rst

```diff
@@ -385,6 +385,64 @@ extra_sections: <list of extra binary sections>
     extra, unexpected sections in the Zephyr binary unless they are named
     here. They will not be included in the size calculation.
 
+harness: <list of harness>
+    A set of string harness for the testcase. This keyword means that a
```

Comment:
Not sure what "a set of string harness" is... perhaps merge these two sentences into:

    A list of harnesses needed to run the tests successfully.

Reply:
Addressed. Kindly check.